src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-gray;
```

Comment:
Ensure that the variable `$color-gray` is defined in your SCSS files or imported correctly. Otherwise, it will cause a compilation error.